lib/utils.js

```diff
@@ -35,17 +36,26 @@ exports.basicAuth = function(username, password) {
 
 exports.findAvailablePort = function(app){
 
-  var port = (process.env.PORT || 3000);
+  var port = null;
+
+  try {
+    port = Number(fs.readFileSync(__dirname+'/../port.tmp'));
```

Comment:
I wonder whether this should be in a `tmp` directory? or else `.port.tmp` so it's hidden?